CBLClient/Apps/CBLTestServer-Dotnet/TestServer/PredictiveQueriesMethods.cs

```diff
@@ -0,0 +1,208 @@
+// 
+//  DocumentMethods.cs
+// 
+//  Author:
+//   Sridevi Saragadam  <sridevi.saragadam@couchbase.com>
+// 
+//  Copyright (c) 2017 Couchbase, Inc All rights reserved.
+// 
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+// 
+//  http://www.apache.org/licenses/LICENSE-2.0
+// 
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// 
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Net;
+
+using JetBrains.Annotations;
+
+using Couchbase.Lite.Enterprise.Query;
+
+using static Couchbase.Lite.Testing.DatabaseMethods;
+using static Couchbase.Lite.Query.QueryBuilder;
+using Couchbase.Lite.Util;
+using Couchbase.Lite.Query;
+
+namespace Couchbase.Lite.Testing
+{
+    internal static class PredictiveQueriesMethods
+    {
+        #region Public Methods
+        public static void RegisterModel([NotNull] NameValueCollection args,
+                                  [NotNull] IReadOnlyDictionary<string, object> postBody,
+                                  [NotNull] HttpListenerResponse response)
+        {
+            var modelName = postBody["model_name"].ToString();
```

Comment:
if variable is of type string than use string as object type instead of var.

Reply:
actually all other apis for other files are written with generic value. I do not see specific datatype defined. Anyways, changed to String